cost/cloud_cost_anomaly_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v2.4
+
+- adding an input list for a mix of dimensions
+- validate each input from the user input list, if all are valid dimensions, it will runs anomaly report. Otherwise when user inputs at least one invalid dimension, policy will stop their processes, escalating to email you invalid/valid dimension you used
```

Comment:
Can we change this to "Validate each input from the user input list, if all are valid dimensions, it will run an anomaly report. Otherwise, when the user inputs at least one invalid dimension, policy will fail, escalating to emailing you about the invalid or valid dimension you used."

Reply:
Coo